src/structures/game_session.ts

```diff
@@ -741,6 +754,7 @@ export default class GameSession {
         if (multipleChoiceMode && this.gameRound.incorrectMCGuessers.has(userID)) return 0;
         if (this.gameType !== GameType.ELIMINATION) {
             this.participants.add(userID);
+            this.ownerQueue.push(userID);
```

Comment:
Won't this add the same person multiple times

Reply:
Whoops, you're right. I'll add in a `includes` check before